cmd/up.go

```diff
@@ -29,19 +29,27 @@ import (
 
 	"github.com/spf13/cobra"
 	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
 // ReconnectingMessage is the message shown when we are trying to reconnect
 const ReconnectingMessage = "Trying to reconnect to your cluster. File synchronization will automatically resume when the connection improves."
 
+var (
+	localClusters = []string{"127.", "10.", "172.", "192.", "169.", "localhost"}
```

Comment:
I'd remove `10.` from this list, since that it's also a typical IP for a private network (that's the default for GKE, EKS, Digital Ocean, etc...), and for the case where you access kubernetes through a VPN instead of through the public internet.

Reply:
Also, could the address of a cluster be an ipv6 address? This might be useful: https://stackoverflow.com/questions/41240761/go-check-if-ip-address-is-in-private-network-space